tools/integration_tests/readonly/readonly_test.go

```diff
@@ -66,6 +67,11 @@ func TestMain(m *testing.M) {
 
 	flags := [][]string{{"--o=ro", "--implicit-dirs=true"}, {"--file-mode=544", "--dir-mode=544", "--implicit-dirs=true"}}
 
+	if setup.TestBucket() == "" && setup.MountedDirectory() != "" {
+		log.Printf("Please pass --testBucket mounted in mountedDirectory.")
```

Comment:
nit: Please pass the name of bucket mounted at mountedDirectory to --testBucket flag.